r2dbc-mysql/src/main/java/io/asyncer/r2dbc/mysql/MySqlConnectionFactoryProvider.java

```diff
@@ -330,6 +330,17 @@ public final class MySqlConnectionFactoryProvider implements ConnectionFactoryPr
      */
     public static final Option<Boolean> METRICS = Option.valueOf("metrics");
 
+    /**
+     * Since the MySQL server silently converts BIT to TINYINT(1) when creating tables,
+     * should the driver treat the datatype TINYINT(1) as the BIT type?
+     * <p>
+     * Note: If {@code tinyInt1isBit=true}, TINYINT(1) columns, whether SIGNED or UNSIGNED,
+     * will be represented as {@link Boolean} by default.
+     *
+     * @since 1.4.0
+     */
```

Comment:
```suggestion
    /**
     * Option to whether the driver should interpret MySQL's TINYINT(1) as a BIT type.
     * When enabled, TINYINT(1) columns (both SIGNED and UNSIGNED) will be treated as
     * BIT.
     * 
     * @since 1.4.0
     */
```